main/src/main/scala/sbt/internal/LibraryManagement.scala

```diff
@@ -307,17 +307,11 @@ private[sbt] object LibraryManagement {
   def withExcludes(out: File, classifiers: Seq[String], lock: xsbti.GlobalLock)(
       f: Map[ModuleID, Vector[ConfigRef]] => UpdateReport
   ): UpdateReport = {
-    import sjsonnew.shaded.scalajson.ast.unsafe.JValue
     import sbt.librarymanagement.LibraryManagementCodec._
     import sbt.util.FileBasedStore
-    implicit val isoString: sjsonnew.IsoString[JValue] =
-      sjsonnew.IsoString.iso(
-        sjsonnew.support.scalajson.unsafe.CompactPrinter.apply,
-        sjsonnew.support.scalajson.unsafe.Parser.parseUnsafe
-      )
     val exclName = "exclude_classifiers"
     val file = out / exclName
-    val store = new FileBasedStore(file, sjsonnew.support.scalajson.unsafe.Converter)
+    val store = new FileBasedStore(file)
```

Comment:
Do you see difference in load time from this change?

Reply:
I havent benchmarked this change against my sample project in #5508...

Do you think I should add that sample project to a scripted test so is easier to check? or is it too much for a unit test?

Reply:
I think it's ok for the tests to prove correctness, but if we're making some optimization or changes aimed at performance improvement, I would like to see some data even if it's casually done local testing.

Reply:
Hi @eed3si9n! I rebased this PR and compared the performance of this PR versus 1.4.0-M1 and did not see any significant improvements. I can try to write some jmh benchmarks, but I see there aren't benchmarks in the project...

WDYT?

Reply:
I have a mixed feeling about jmh. In general they are useful. But since the Scala compiler is complicated, even with 10 warmup or whatever C2 would keep optimizing for like 10 minutes, even longer.. so that means that more iteration you run, the performance gets better at least for compilation. See https://github.com/sbt/zinc/pull/767#issuecomment-621935005. This might affect sbt indirectly since there are some aspect of compilation. So to minimize the fluctuation, we should reuse `*.class` files between the iteration?

Existing code that might be relevant for benchmarking is scripted test and server test.
- https://github.com/sbt/sbt/blob/develop/scripted-sbt-redux/src/main/scala/sbt/scriptedtest/ScriptedTests.scala
- https://github.com/sbt/sbt/blob/develop/scripted-sbt-redux/src/main/scala/sbt/scriptedtest/RemoteSbtCreator.scala
- https://github.com/sbt/sbt/blob/develop/server-test/src/test/scala/testpkg/TestServer.scala#L173-L174

See also https://github.com/eatkins/scala-build-watch-performance